doc/source/whatsnew/v2.2.0.rst

```diff
@@ -28,7 +28,7 @@ enhancement2
 
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
--
+- Added ``calamine`` as an engine to ``read_excel`` (:issue:`50395`)
```

Comment:
This seems significant to me, I'd be in favor of a section in the top `Enhancements`.

Reply:
Something like? When odfpy and pyxlsb were added, their news was put in "Other enhancements".

```rst
Calamine engine for :func:`read_excel`
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The ``calamine`` engine was added to :func:`read_excel`.
It's used 'python-calamine', that binging for Rust library `calamine <https://crates.io/crates/calamine>`__.
This engine supports Excel file (``.xlsx``, ``.xlsm``, ``.xls``, ``.xlsb``) and OpenDocument spreadsheets (``.ods``).

There are two advantages of this engine:

1. Calamine is faster than Python engines, some benchmarks show results up to 5x faster then 'openpyxl', 20x - 'odf', 4x - 'pyxlsb', and 1.5x - 'xlrd'.
   But, 'openpyxl' and 'pyxlsb' are faster in reading a few rows from large files because of lazy iteration over rows.
2. Calamine supports the recognition of datetime in ``.xlsb`` files.

.. code-block:: python

   pd.read_excel("path_to_file.xlsb", engine="calamine")


For more, see :ref:`io.calamine` in the user guide on IO tools.
```

Reply:
Yes, it's true. Fixed.